nl/nl_linux.go

```diff
@@ -362,46 +362,44 @@ func (a *RtAttr) Serialize() []byte {
 
 type NetlinkRequest struct {
```

Comment:
This is an exported type. Other project may already be using it and the members you are removing are unfortunately exported as well.